core/src/main/scala/cats/mtl/syntax/chronicle.scala

```diff
@@ -30,18 +20,34 @@ final class ChronicleOps[F[_], A](val fa: F[A]) extends AnyVal {
   }
 
   def absolve[E](a: => A)(implicit chronicle: MonadChronicle[F, E]): F[A] = {
-    chronicle.absolve(a, fa)
+    chronicle.absolve(fa)(a)
   }
 
   def condemn[E](implicit chronicle: MonadChronicle[F, E]): F[A] = {
     chronicle.condemn(fa)
   }
 
   def retcon[E](cc: E => E)(implicit chronicle: MonadChronicle[F, E]): F[A] = {
-    chronicle.retcon(cc, fa)
+    chronicle.retcon(fa)(cc)
   }
+}
+
+final class ChronicleIdOps[E](val e: E) extends AnyVal {
+  def dictate[F[_]](implicit chronicle: MonadChronicle[F, E]): F[Unit] = {
+      chronicle.dictate(e)
+    }
+
+  def disclose[F[_], A](implicit chronicle: MonadChronicle[F, E], monoid: Monoid[A]): F[A] = {
```

Comment:
Originally `disclose` needs `Default` (or cats' `Empty`) intance:
https://hackage.haskell.org/package/these-0.7.5/docs/Control-Monad-Chronicle.html#v:disclose

Reply:
Changing `Monoid` bound to `Empty` only adds `alleycats` dependency, but doesn't break the existing code, since `Empty` is derived from `Monoid`.

Reply:
I think adding an `alleycats` dependency is a non-starter personally. I think it's fine to break with Haskell in this case. :)